### DISCUSSION TITLE
directus_users access through API

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#13024] [Archive]
Could someone explain me why with
`http://localhost:8055/items/directus_users?access_token=4b5e6cf5&fields=last_name`
_where 4b5e6cf5 is my Admin access token (working with /server/info and the collections i created)_
i get a `FORBIDDEN`?

**Comment by rijkvanzanten** [comment]
Because the system collections are accessed through different endpoints. In this case `/users` instead of `/items/users` 👍🏻
